<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFontSize</class>
 <widget class="QWidget" name="SetFontSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>32</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>32</width>
    <height>168</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32</width>
    <height>168</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
